src/commcare_cloud/ansible/roles/formplayer/templates/application.properties.j2

```diff
@@ -16,7 +16,7 @@ sentry.dsn={{ formplayer_sentry_dsn }}
 {% endif %}
 sentry.environment={{ env_monitoring_id }}
 sentry.tags.HQHost={{ host }}
-sentry.release={{ formplayer_current_release_name if _should_update_formplayer_in_place else formplayer_release_name }}
+sentry.release={{ current_formplayer_release }}
```

Comment:
Prior to https://github.com/dimagi/commcare-cloud/pull/6395 this was `formplayer_release_name`. I assume it is true, but will the new `current_formplayer_release` variable always be set when this template is rendered? In other words, is this template used in any other context, particularly ones where  `current_formplayer_release` is not set?

Reply:
Good point. We only reference the application.properties.j2 template in the task that also sets the `current_formplayer_release` var. I don't know if it is better to default to `formplayer_release_name` in the template if `current_formplayer_release` isn't defined, or just leave this as is so that any future attempt at rendering this file without the var defined will fail hard. I am leaning towards the latter. What do you think?

Reply:
I agree, the latter seems better. I was concerned about potential other usages, but it sounds like there are none so that's fine.